catroid/src/org/catrobat/catroid/formulaeditor/FormulaEditorEditText.java

```diff
@@ -40,12 +41,13 @@
 public class FormulaEditorEditText extends EditText implements OnTouchListener {
 
 	private static final BackgroundColorSpan COLOR_ERROR = new BackgroundColorSpan(0xFFF00000);
-	private static final BackgroundColorSpan COLOR_HIGHLIGHT = new BackgroundColorSpan(0xFFFFFF00);
+	private static final BackgroundColorSpan COLOR_HIGHLIGHT = new BackgroundColorSpan(0xff33b5e5);
```

Comment:
Have we decided on whether to use uppercase or lowercase for hex values? I think we use only uppercase in the `colors.xml`, at least I advocate on doing so.